### DISCUSSION TITLE
Can I create AcroFields and Signature fields using QuestPDF?

### DISCUSSION THREAD

**Discussion by donat-comparit** [QuestPDF/QuestPDF#703] [Q&A] [answered]
Hi, 

I would like to know if QuestPDF has already support for generating Acrofields and Forms. Also, I need to know if I can add Signature fields into the PDF. I can't see anything about these topics in the documentation so I suppose they are not supported yet, but are they on the timetable of QuestPDF and if so when will they be implemented? That is for me a deal breaker.

**Comment by girlpunk** [comment]
Unfortunately Acroforms are not available at this moment. However, it is possible to generate content to visually resemble forms (e.g. checkboxes, text fields).

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Both features remain on our backlog. Interactive form fields (AcroForms) are tracked in [#1189](https://github.com/QuestPDF/QuestPDF/issues/1189), and digital signature support is tracked in [#1193](https://github.com/QuestPDF/QuestPDF/issues/1193) — feel free to follow and upvote those to help us gauge demand. In the meantime, form-like content can be reproduced visually (checkboxes, text fields, etc.), and a common approach for signatures is to generate the PDF with QuestPDF and then sign it using a dedicated signing library.

Should you encounter any further problems, please don't hesitate to open a new issue.
Thank you!
